Sources/SwiftDocC/Infrastructure/DocumentationContext+Breadcrumbs.swift

```diff
@@ -38,7 +38,7 @@ extension DocumentationContext {
             // To match the caller's expectations we remove the starting point and then flip the paths.
             .map { $0.dropFirst().reversed() }
             .sorted { (lhs, rhs) -> Bool in
-                // Order a path rooted in a technology as the canonical one.
+                // Order a path rooted in a tutorial table-of-contents as the canonical one.
                 if options.contains(.preferTechnologyRoot), let first = lhs.first {
```

Comment:
Not for this PR, but at some point in the future we should rename the options too, so that our terminology is consistent.

Reply:
this naming is even worse because "technology root" is something different from a "technology" page 🤦

Reply:
Updated in https://github.com/swiftlang/swift-docc/pull/1039/commits/d8799080be42f07b441692ddbe7cb5a0722eb83c